tools/importer-msgraph-metadata/go.mod

```diff
@@ -71,3 +71,5 @@ require (
 	gopkg.in/yaml.v2 v2.4.0 // indirect
 	gopkg.in/yaml.v3 v3.0.1 // indirect
 )
+
+replace github.com/hashicorp/pandora/tools/sdk => ../sdk
```

Comment:
Are we supposed to be doing this replacement in main as well?

Reply:
Yep this is intentional - it ensures the `importer-msgraph-metadata` tool aligns with the other tools which already include the replace - which means that the `sdk` directory is sourced from the local file system, rather than [this commit (vendored as `v0.0.0-20230809001200-97c549958463` in the `go.mod`)](https://github.com/hashicorp/pandora/tree/97c5499584637328cfda7ee169668f9286fa6859) which it's doing today.